src/flows/Onboarding/hooks.ts

```diff
@@ -72,6 +74,36 @@ const useEmployment = (employmentId: string | undefined) => {
   });
 };
 
+const useCompany = (companyId: string) => {
```

Comment:
The `hooks.ts` is getting bigger an bigger. I suggest we have in this file only the `useOnboarding` hook and move API hooks to a new file `onboarding/api.ts`. We should then have the same pattern for the all the other flows (in a separate PR). Wdyt?